src/plugins/filter/demo/post-filter.js

```diff
@@ -0,0 +1,17 @@
+( function() {
+window[ "wb-filter" ] = {
+	filterCallback: function( $field, $elm, settings ) {
+		var $sections =	$elm.find( "section" ),
```

Comment:
I suppose we can have one or more default ones but I'm not sure it will be that simple in many cases. Because this can target any content using selectors. I'm trying to be as least opinionated as possible.